config/param-validation.js

```diff
@@ -43,7 +19,14 @@ export default {
   login: {
     body: {
       username: Joi.string().required(),
-      password: Joi.string().required()
+      password: Joi.string().required(),
+      // Should be required once mobile apps get updated:
+      // TODO: consider adding versioning to API so we can roll
+      // out new features to different clients.
+      name: Joi.string(),
+      bio: Joi.string(),
```

Comment:
Why are bio and website parameters for login?

Reply:
Ah, nice catch.